org.gnu.liquidwar6.yaml

```diff
@@ -0,0 +1,142 @@
+
+  - name: libcaca
+    config-opts:
+      - --disable-doc
+      - --disable-python
+      - --disable-ruby
+      - --disable-static
+    sources:
+      - type: archive
+        url: http://caca.zoy.org/files/libcaca/libcaca-0.99.beta19.tar.gz
+        sha256: 128b467c4ed03264c187405172a4e83049342cc8cc2f655f53a2d0ee9d3772f4
+
+  - name: graphviz
+    buildsystem: autotools
+    no-autogen: true
+    config-opts:
+      - --without-x
+      - --with-qt=no
+      - --with-gtk=no
+      - --enable-swig=no
+      - --with-webp=no
+      - --with-rsvg=no
+      - --with-visio=no
+      - --with-gdk-pixbuf=no
+      - --with-pangocairo=no
+    cleanup:
+      - /share/man
+      - /share/graphviz
+    sources:
+      - type: archive
+        url: https://gitlab.com/api/v4/projects/4207231/packages/generic/graphviz-releases/8.0.3/graphviz-8.0.3.tar.gz
+        sha256: 0cbc80423fe6582d5e084f0fb86159ef90c7d6e4f7ab2b9bf53cd98ae1c2b507
+
+  - name: cunit
+    buildsystem: autotools
+    config-opts:
+      - --disable-static
+      - --enable-automated
+      - --enable-basic
+      - --enable-console
+      - --enable-curses
+      - --disable-examples
+      - --disable-test
+    cleanup:
+      - /include/CUnit
+      - /lib/pkgconfig/
+    sources:
+      - type: archive
+        url: https://download.sourceforge.net/cunit/CUnit-2.1-3.tar.bz2
+        sha256: f5b29137f845bb08b77ec60584fdb728b4e58f1023e6f249a464efa49a40f214
+      - type: shell
+        commands:
+          - autoreconf -fi
+
+  - name: csound
+    buildsystem: cmake
+    build-options:
+      env:
+        PYTHONHOME: /app
+        PYTHONPATH: /usr/lib/python3.10
+      arch:
+        arm:
+          config-opts:
+          - -DHAVE_NEON=OFF
+          - -DCMAKE_CXX_FLAGS=-DPFFFT_SIMD_DISABLE
+          - -DCMAKE_C_FLAGS=-DPFFFT_SIMD_DISABLE
+    sources:
+    - type: archive
+      url: https://github.com/csound/csound/archive/refs/tags/6.18.1.tar.gz
+      sha256: b84be8237fa9258ef60fdddf36fae27b20c11665811bf5ed12540da9a9a4414e
+
+  - name: liquidwar6
+    buildsystem: autotools
+    config-opts:
+      - --enable-allinone
+      - --disable-static
+      - --disable-rpath
+    build-options:
+      cflags: -Wno-error=deprecated-declarations -Wno-error=address -Wno-error=use-after-free
+    post-install:
+      - install -D /app/share/pixmaps/liquidwar6.png /app/share/icons/hicolor/128x128/apps/liquidwar6.png
+      - install -Dm644 org.gnu.liquidwar6.metainfo.xml /app/share/metainfo/org.gnu.liquidwar6.metainfo.xml
+    sources:
+      - type: archive
+        url: http://ftp.gnu.org/gnu/liquidwar6/liquidwar6-0.6.3902.tar.gz
+        sha256: d605549afae403439524e572322e45ebf5b6db8ea59505e5d51cb581a8b5e6a4
+      - type: shell
+        commands:
+          - autoreconf -ivf
+      - type: archive
+        url: http://download.savannah.gnu.org/releases/liquidwar6/0.6.3902/liquidwar6-extra-maps-0.6.3902.zip
```

Comment:
are you should this doesn't need to be extracted in a separate directory inside the build tree?

Reply:
You are right. This was bogus. Well spotted.